snyk/cli
# feat: meaningful auth errors cli 575

## Pull Request Submission Checklist

## What does this PR do?

## Where should the reviewer start?

## How should this be manually tested?

- Clear your config using the `snyk config clear` command
- Build and run any of the `code test` or `test` command and you should get a nice error message with an Error Catalog code
- You can now authenticate using the CLI and disable the Snyk Code feature in your org settings on the Snyk app website.  Running the `code test` command now should yield a better feature not enabled message for Snyk Code.


## What are the relevant tickets?
- [CLI-575](https://snyksec.atlassian.net/browse/CLI-575)

[CLI-575]: https://snyksec.atlassian.net/browse/CLI-575?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
